### ISSUE TITLE
🐛[BUG] The Wyrmking Descends 

### ISSUE THREAD

**Issue by Squinkerdoodle** [HorizonFFXI/HorizonXI-Issues#109]
**What does this bug affect?**
Bahamut spams Megaflare upon reaching 89%-90% HP. When he should do the first Megaflare he will continue to use it repeatedly after the first one is done. Additionally I went ahead and beat the fight, and no loot was given after before or after being ported out of the fight arena.

**Steps to Reproduce**
Go into the fight and lower Bahamut's HP to 89%-90%. Kill Bahamut and see if loot is given.

**Expected behavior**
Bahamut should only use one Megaflare @ 90%. Loot should be awarded after beating Bahamut.

**Setup Information**
Version 30220906_0

**Additional context**
Basically seems to be the same issue as reported in https://github.com/HorizonFFXI/HorizonXI-Issues/issues/107

**Comment by dallano** [comment]
@CriticalXI Do you want me to take care of the bugs for these battlefields you brought over? I imagine you're busy elsewhere.

**Comment by dallano** [comment]
https://github.com/AirSkyBoat/AirSkyBoat/pulls fix!
